samples/snippets/csharp/VS_Snippets_Remoting/NCLPhysicalAddress/CS/NCLPhysicalAddress.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
@dotnet/docs do we allow adding csproj files to this location?

Reply:
it would be nice to have guidance. I added the csproj to be able to run the example. It should be easy to run the example so I think it worth to have it part of the examples.

Reply:
We expect the /samples/snippets folder has any samples. We've turned on a CI system in dotnet/docs, and we'll be adding it in this repo shortly. Once it's turned on, any PR that updates sample code will be run through the CI build.